container-compose-staging.yml

```diff
@@ -4,6 +4,7 @@ services:
       context: extenginx
       additional_contexts:
         - nginx_snippet_source=./nginx_snippets
+        - nginx_webroot_content=./kdlp.underground.software
```

Comment:
but you're still bindmouting the webroot here?

Reply:
only in dev yes